topics/galaxy-project/slides/introduction.html

```diff
@@ -0,0 +1,172 @@
+---
+layout: tutorial_slides
+logo: assets/images/Galrollup.png
+title: "A 5 minute introduction to the Galaxy project"
+video: true
+time_estimation: 5 Minutes
+key_points:
+  - "To summarise, Galaxy is a global open science collaboration"
+  - "Providing accessible, shareable and reproducible analyses for researchers"
+  - "Embedded in the much bigger open source and open science ecosystems"
+  - "New contributors and investigators are highly valued and welcomed"
+  - "Related lessons in this topic describe details of major project structures and stories of community development activities"
+  - "To guide participants finding ways to join in our work"
+contributors:
+  - ggsc
+  - fubar2
```

Comment:
If you like can use our fancy new contributions syntax to more accurately record contributions to a training material, declaring which people should be recorded for authorship or just editing.

```
contributions:
  authorship:
    - shiltemann
  editing:
    - bebatut
    - hexylena
  funding:
    - carpentries
  testing:
    - userX
  ux:
    - userY
  infrastructure:
    - userZ
```

see the [full gtn tutorial schema documentation](https://training.galaxyproject.org/training-material/topics/contributing/tutorials/schemas/tutorial.html) for details on the individual roles.